fieldservice_delivery/models/fsm_order.py

```diff
@@ -7,4 +7,25 @@
 class FSMOrder(models.Model):
     _inherit = 'fsm.order'
 
-    carrier_id = fields.Many2one('delivery.carrier', string="Delivery Method")
+    carrier_id = fields.Many2one('delivery.carrier', string="Delivery Method",
+                                 compute="_compute_carrier_id")
+
+    #Note: This function only works if the user has configured their
+    #      Delivery Methods to have 'Fedex Ground', 'Fedex Standard-
+    #      Overnight', and 'Fedex Priority-Overnight'
+    def _compute_carrier_id(self):
+        if self.priority == '0' or self.priority == '1':
+            fedex = self.env['delivery.carrier'].\
+                search([('name', '=', 'Fedex Ground')])
+            if fedex:
+                self.carrier_id = fedex
+        elif self.priority == '2':
+            fedex = self.env['delivery.carrier'].\
+                search([('name', '=', 'Fedex Standard-Overnight')])
+            if fedex:
+                self.carrier_id = fedex
+        elif self.priority == '3':
+            fedex = self.env['delivery.carrier'].\
+                search([('name', '=', 'Fedex Priority-Overnight')])
+            if fedex:
+                self.carrier_id = fedex
```

Comment:
This should go in a separate module in the customer repo. Anyone can have a different policy.